### PR TITLE
First iteration 03-2024 Diāna Roze

### PR DESCRIPTION
**Pull request by dhrtiradha**
I managed to make all the tests green, though the 4th test is not properly working during the second round of tests. I suppose that either the data needs to be cleared or the search for airports needs to be more precise.
Maybe you will see some more mistakes?

### CODE REVIEW

File: WebApplicationFP2/Storage/FlightStorage.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        {
+            _flights.Clear();
+        }
+
+        public static void DeleteFlight(int id)
+        {
+            _flights.RemoveAll(f => f.Id == id); 
+        }
+
+        //public static bool FlightExists(Flight flight)
```
Comment (EdgarsSalmins) [inline WebApplicationFP2/Storage/FlightStorage.cs:29]: kapēc šo nelieto?

File: WebApplicationFP2/Controllers/AdminController.cs [hunk 2]
```diff
@@ -9,11 +11,78 @@ namespace WebApplicationFP2.Controllers
...
+                    { 
+                        return BadRequest("Arrival time must be after departure time.");
+                    }
+                }
+                else 
+                { 
+                    return BadRequest("Invalid date format.");
+                }
+                
+                var flights = FlightStorage.GetAllFlights();
```
Comment (EdgarsSalmins) [inline WebApplicationFP2/Controllers/AdminController.cs:54]: šo jau labāk meklēt pašā storage. :) un atgriež true/false.

File: WebApplicationFP2/Controllers/AdminController.cs [hunk 3]
```diff
@@ -9,11 +11,78 @@ namespace WebApplicationFP2.Controllers
...
+                    return Created("", addedFlight);
+                }
+            }
+        }
+
+        [HttpDelete]
+        [Route("flights/{id}")]
+        public IActionResult DeleteFlight(int id)
+        {
+
```
Comment (EdgarsSalmins) [inline WebApplicationFP2/Controllers/AdminController.cs:82]: liekas tukšas rindas.

**Comment by EdgarsSalmins** [comment]
> I managed to make all the tests green, though the 4th test is not properly working during the second round of tests. I suppose that either the data needs to be cleared or the search for airports needs to be more precise. Maybe you will see some more mistakes?

data is being cleared before every test.
image

**Review by EdgarsSalmins — APPROVED** [review]
minimāli sīkumi kādi ir ...bet pirmajā iterācija tas nav būtiski. :) vari pat nelabot.. ) nākošajās iterācijās izlabosies.

File: WebApplicationFP2/Storage/FlightStorage.cs (hunk 1 above)
Comment (dhrtiradha) [reply WebApplicationFP2/Storage/FlightStorage.cs:29]: tagad precizi nepateikšu bet pamatots iemesls bija :D vai nu pedejais tests uz to lamajas vai kaut kas tāds un tāpēc citādi darīju

**Comment by dhrtiradha** [comment]
labi, sapratu, paldies Jums!

**Pull request merged**